src/integrations/gitlens/gitlens.d.ts

```diff
@@ -20,17 +19,16 @@ export interface CreatePullRequestActionContext {
 		readonly isRemote: boolean;
 	};
 	readonly remote:
-		|	{
+		| {
```

Comment:
tslint warning from prettier again - maybe next iteration we can run prettier on everything and add some kind of enforcement for it